docker-compose-services/chromedriver/phpunit.xml.dist

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is this required?

Reply:
Not really, its just an example of how you might want to empty out your phphunit.xml.dist for functional javascript. But it also might be confusing. I'm happy to remove it.

Reply:
Do we want to provide any behat setup examples or not?

Reply:
None have surfaced.